packages/domain/src/use-cases/publications/types.ts

```diff
@@ -49,13 +54,17 @@ export enum CollectPolicyType {
 
 export type ChargeCollectPolicy = {
   type: CollectPolicyType.CHARGE;
+  followersOnly: boolean;
   fee: Amount<Erc20>;
-  recipient: EthereumAddress;
   metadata: NftMetadata;
+  recipient: EthereumAddress;
   mirrorReward: number;
-  collectLimit?: number;
   timeLimited: boolean;
-  followersOnly: boolean;
+  collectLimit?: number;
+  recipients?: RecipientWithSplit[]; // multirecipientFeeCollectModule
+  endTimestamp?: number;
+  vault?: EthereumAddress; // erc4626FeeCollectModule
+  depositToAave?: boolean; // aaveFeeCollectModule
```

Comment:
`recipients`, `endTimestamp`, `vault`, and `depositToAave` are new fields to determine which collect module to use